### ISSUE TITLE
Config Flow error

### ISSUE THREAD

**Issue by fusion94** [natekspencer/hacs-govee_ble#11]
Looks like this might no longer be maintained but after installing and trying to locate a H5174 I'm getting this error:

`Config flow could not be loaded: 500 Internal Server Error Server got itself in trouble`

**Comment by natekspencer** [comment]
Yeah the new Bluetooth and Govee core integrations introduced in HA a few months back cover most of what this was doing so no other updates will be made.

**Comment by spokenmusic** [comment]
Hi! I just tried to use this and its giving me the same error stated as on top.

(I'm using a virtual machine with VirtualBox on linux for home assistant)

**Comment by jq-c0** [comment]
Having the same issue. Looking to connect to a light strip.
